.gitignore

```diff
@@ -1,3 +1,4 @@
+zig-cache/
```

Comment:
Added this back because it's nice to be able to switch Zig versions without these files appearing in staging.